<launch>
  <arg name="plan_file_short_hand" default="debug_points.yaml" />
  <arg name="plan_file" default="$(find agv_explore)/param/$(arg plan_file_short_hand)" />

  <arg name="use_tvoc" default="true" />

  <!-- convert tvoc/eco2 to gas node -->
  <node pkg="agv_explore" type="get_tvoc.py" name="get_tvoc" if="$(arg use_tvoc)" />

  <!-- gas mapping -->
  <node pkg="agv_explore" type="gas_mapping_tf.py" name="gas_mapping">
    <rosparam file="$(find gas_distribution)/param/gas_visual_param.yaml" command="load" />
  </node>

  <!-- launch gas_patrol node -->
  <node pkg="agv_explore" type="gas_patrol.py" name="gas_patrol" output="screen">
    <rosparam command="load" file="$(arg plan_file)" />
  </node>
  <node pkg="agv_explore" type="gas_scrutinize.py" name="gas_scrutinize" output="screen"/>

</launch>
